examples/custom-dns-configuration-cluster-aws/customDnsConfigurationClusterAws.json

```diff
@@ -0,0 +1,30 @@
+{
+  "AWSTemplateFormatVersion" : "2010-09-09",
+  "Description" : "This template creates a Project on the MongoDB Atlas API, this will be billed to your Atlas account.",
```

Comment:
This looks like Example for Project Resource. Can you refresh for custom DNS Configuration Cluster AWS